ui/de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -654,6 +655,11 @@ void restoreWindows(List<WindowState> windowStates) {
         for (WindowState ws : windowStates) {
             desktopWindowManager.show(ws);
         }
+        if (userInfo.hasAgaveRedirect()) {
+            AgaveAuthPrompt prompt = AgaveAuthPrompt.getInstance();
+            prompt.show();
+            desktopWindowManager.stickWindowToTop(prompt);
```

Comment:
So does this add a 4th case where the prompt will show?
Since the prompt is already shown when the bootstrap endpoint returns, and the sessions load after the bootstrap call returns.
Could this `stickWindowToTop` call happen when the prompt is shown in the bootstrap callback, instead?